app/build.gradle.kts

```diff
@@ -5,30 +5,16 @@ plugins {
 
 android {
     namespace = "com.scottyab.rootbeer.sample"
-    compileSdk =
-        libs.versions.android.compile.sdk
-            .get()
-            .toInt()
-    buildToolsVersion =
-        libs.versions.android.build.tools
-            .get()
 
     defaultConfig {
         applicationId = "com.scottyab.rootbeer.sample"
-        minSdk =
-            libs.versions.android.min.sdk
-                .get()
-                .toInt()
-        targetSdk =
-            libs.versions.android.target.sdk
-                .get()
-                .toInt()
-        versionName = findProperty("VERSION_NAME").toString()
+        versionName = version.toString()
```

Comment:
`Project.version` already has the correct value set (as set from the root `build.gradle.kts`) so this can be referenced now instead of re-referencing the property.